config/settings/test.py

```diff
@@ -60,3 +62,16 @@
         'django.template.loaders.app_directories.Loader',
     ]),
 ]
+DATABASE = {
+	'default':{
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'TUTERIATEST',
+        'USER': 'beast',
+        'PASSWORD': '',
+        'HOST': '',
+        'PORT': '',
+        }
+	}
+
```

Comment:
The Database environmental variable has already been set in the `common.py` module. you can see it is imported at the top. To setup your development environment, all you needed to do was to look at the **Database** section in the readme

    Create a postgres database and setup the environmental variable DATABASE_URL to point to the url format of the database created.

and when django loads, it would read the DATABASE settings from the environmental variable.

No need to manually specify a configuration in the `local.py` file.